### ISSUE TITLE
The `--with-symlink-targets` option for the command `ls` no longer works

### ISSUE THREAD

**Issue by JosephTLyons** [nushell/nushell#1299]
**Describe the bug**
When using the option `--with-symlink-targets` for the `ls` command, the `targets` column isn't being displayed.  It is still viewable when using `ls --full`.

**To Reproduce**
1. run `ls --with-symlink-targets` and notice that the `targets` column is missing

**Expected behavior**
I would expect to the see the `targets` columns

**Configuration (please complete the following information):**
 - OS: macOS 10.15.2
 - Version 0.8.0

Running the `dbg!()` macro on the variable `with_symlink_targets` in `files.rs` right before the code block that inserts the column, I can see that the value is coming in as `false`, so something must be either tampering with it prior to that point or it isn't being properly set.  I feel I should be able to figure it out, but since I'm not familiar with the codebase too much, it could take a bit longer than normal.

**Comment by JosephTLyons** [comment]
https://github.com/nushell/nushell/pull/1300

**Comment by JosephTLyons** [comment]
The more I think about it, I'm not sure `--with-symlink-targets` is even necessary.  `--full` is a shorter command and I know that the philosophy is sort of to keep the amount of options low, so it seems a bit out of place to have one command show one additional column while `--full` already shows them all.  The other options on `ls` really just tweak the output of existing columns, where `--with-symlink-targets` does exactly what `--full` does, but with less information.  Opinions?

**Comment by jonathandturner** [comment]
@JosephTLyons - Right now we're in a bit of "exploration" mode for Nu.  It's not quite clear what the right balance of arguments vs aliases will be once we added aliases, so we're interested and just trying things out and seeing what feels right.

We're definitely planning on doing a full survey of what we've built at some point to see what feels the most natural.

I think we can leave the flag in for now, but I agree that once aliases come we might decide it's better to put it in --full and then let people configure a pipeline that pulls it out for them.
